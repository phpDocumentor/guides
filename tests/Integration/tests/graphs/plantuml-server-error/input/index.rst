==============
Uml Directive
==============

.. uml:: /Plantuml/something.plantuml
   :align: center
   :caption: Figure 1-1: Application flow
   :width: 1000


.. uml:: Plantuml/something.plantuml
   :align: center
   :caption: Figure 1-1: Application flow
   :width: 1000
