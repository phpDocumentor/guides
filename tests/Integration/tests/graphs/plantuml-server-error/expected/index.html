<!-- content start -->
    <div class="section" id="uml-directive">
            <h1>Uml Directive</h1>
            <figure class="uml-diagram" style="width: 1000">
                <figcaption>Figure 1-1: Application flow</figcaption>
            </figure>
            <figure class="uml-diagram" style="width: 1000">
                <figcaption>Figure 1-1: Application flow</figcaption>
            </figure>
    </div>
<!-- content end -->
